src/rules/noShadowedVariableRule.ts

```diff
@@ -31,15 +31,37 @@ class NoShadowedVariableWalker extends Lint.BlockScopeAwareRuleWalker<ScopeInfo,
         return new ScopeInfo();
     }
 
+    public visitBindingElement(node: ts.BindingElement) {
+        // TODO: handle node.dotDotDotToken?
+        var isSingleVariable = node.name.kind === ts.SyntaxKind.Identifier;
```

Comment:
which part? just this one-line check for syntax kind? I disagree; having too many utilities is bad and it's pretty readable here as-is